src/main/java/com/google/devtools/build/lib/buildeventservice/client/BuildEventServiceGrpcClient.java

```diff
@@ -83,6 +99,10 @@ public void publish(PublishLifecycleEventRequest lifecycleEvent)
     try {
       besBlocking
           .withDeadlineAfter(RPC_TIMEOUT.toMillis(), MILLISECONDS)
+          .withInterceptors(
+              TracingMetadataUtils.attachMetadataInterceptor(
+                  TracingMetadataUtils.buildMetadata(
+                      buildRequestId, commandId.toString(), "publish_lifecycle_event", null)))
```

Comment:
nit: add the relevant comment to the null param, same below

Reply:
done